### ISSUE TITLE
pact-broker verification timeouts

### ISSUE THREAD

**Issue by saikrishna321** [pact-foundation/pact-node#136]
## Software versions

-   **OS**: Mac OSX 10.12.6 
-   **Library version**: LATEST
-   **Node Version**: `8.10.0`

## Expected behaviour

I was expecting to verify the contract that has been published to pact-broker.

## Actual behaviour

Timeouts to verify the contract from pact-broker

## Steps to reproduce

[E2E test](https://github.com/pact-foundation/pact-js/blob/master/examples/e2e/test/provider.spec.js)

## Relevant log files

~/Downloads/pact-js-master/examples/e2e » npm run test:provider                                              sai.krishna@HOLBMAC0176

> e2e@1.0.0 test:provider /Users/sai.krishna/Downloads/pact-js-master/examples/e2e
> mocha test/provider.spec.js

Animal Profile Service listening on http://localhost:8081
  Pact Verification
[2018-12-03T16:34:05.527Z]  INFO: pact-node@6.20.0/49650 on HOLBMAC0176: Verifying Pacts.
[2018-12-03T16:34:05.531Z]  INFO: pact-node@6.20.0/49650 on HOLBMAC0176: Verifying Pact Files
[2018-12-03T16:34:08.662Z]  INFO: pact-node@6.20.0/49650 on HOLBMAC0176: Created './standalone/darwin-1.61.1/bin/pact-provider-verifier 'https://test.pact.dius.com.au/pacts/provider/Animal%20Profile%20Service/consumer/Matching%20Service/version/1.0.1543848742' --provider-base-url 'http://localhost:8081' --provider-states-setup-url 'http://localhost:8081/setup' --broker-username 'dXfltyFMgNOFZAxr8io9wJ37iUpY42M' --broker-password 'O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1' --publish-verification-results 'true' --provider-app-version '1.0.0' --custom-provider-header 'Authorization: basic e5e5e5e5e5e5e5'' process with PID: 49652
    1) should validate the expectations of Matching Service

  0 passing (30s)
  1 failing

  1) Pact Verification
       should validate the expectations of Matching Service:
     Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/sai.krishna/Downloads/pact-js-master/examples/e2e/test/provider.spec.js)
      at Test.Runnable._timeoutError (/Users/sai.krishna/Downloads/pact-js-master/examples/e2e/node_modules/mocha/lib/runnable.js:440:10)
      at Timeout.<anonymous> (/Users/sai.krishna/Downloads/pact-js-master/examples/e2e/node_modules/mocha/lib/runnable.js:251:24)
      at ontimeout (timers.js:482:11)
      at tryOnTimeout (timers.js:317:5)
      at Timer.listOnTimeout (timers.js:277:5)

**Comment by saikrishna321** [comment]
@mefellows Thanks for getting back. I'm able to navigate to  https://test.pact.dius.com.au from the browser. 

I tried to verify from outside my office network and was able to do it successfully. Looks like proxy issue. 

1. I was able to publish the contract to pact-broker from office network and it worked fine.

2. I only see an issue when verifying the published pact from office network. 

Is there any proxy settings from publisher I need to set?
